src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/FileStreamResultExecutor.cs

```diff
@@ -47,6 +47,7 @@ public virtual Task ExecuteAsync(ActionContext context, FileStreamResult result)
 
             if (!serveBody)
             {
+                result.FileStream.Dispose();
```

Comment:
Should we put this inside a `using` block? In the event, any other piece of code in this type throws, we would return without disposing the stream.

Reply:
Sure, updated.